test/topology_custom/test_tablets.py

```diff
@@ -23,3 +23,55 @@ async def test_tablet_change_replication_vnode_to_tablets(manager: ManagerClient
     with pytest.raises(InvalidRequest):
         await cql.run_async("ALTER KEYSPACE test WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 1, 'initial_tablets': 1};")
 
+
+@pytest.mark.asyncio
+async def test_tablet_change_replication_strategy(manager: ManagerClient):
+    cfg = {'enable_user_defined_functions': False,
+           'experimental_features': ['tablets', 'consistent-topology-changes']}
+    server = await manager.server_add(config=cfg)
+
+    cql = manager.get_cql()
+    await cql.run_async("CREATE KEYSPACE test WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};")
+    await cql.run_async("ALTER KEYSPACE test WITH replication = {'class': 'NetworkTopologyStrategy'};")
+
+    res = await cql.run_async("DESCRIBE KEYSPACE test")
+    assert "NetworkTopologyStrategy" in res[0].create_statement, "NetworkTopologyStrategy wasn't switched onto"
+
+    res = await cql.run_async("SELECT * FROM system_schema.scylla_keyspaces WHERE keyspace_name = 'test'")
+    assert len(res) == 0, "tablets replication strategy turned on"
+
+
+@pytest.mark.asyncio
+async def test_tablet_default_initialization(manager: ManagerClient):
+    cfg = {'enable_user_defined_functions': False,
+           'experimental_features': ['tablets', 'consistent-topology-changes']}
+    server = await manager.server_add(config=cfg)
+
+    cql = manager.get_cql()
+    await cql.run_async("CREATE KEYSPACE test WITH replication = {'class': 'NetworkTopologyStrategy', 'replication_factor': 1, 'initial_tablets': 1};")
+
+    res = await cql.run_async("SELECT * FROM system_schema.scylla_keyspaces WHERE keyspace_name = 'test'")
+    assert res[0].initial_tablets > 0, "initial_tablets not configured"
```

Comment:
Nitpick: I hope you realize that pytest has a beautiful system for reporting assert failures, so these assert messages that you write here aren't really helpful?

Reply:
I haven't yet seen any test failure messages that would be verbose enough for anybody but the person who write the test. So even if this part fails and the issue is assigned on me, I'll understand what went wrong (not why it went wrong though). Any other person would need to read the test case code anyway.

But if you have some better suggestion on the message, I will be happy to apply it

Reply:
No message - not even the high-quality "initial_tablets not configured" - can be used by anyone without going back to the test code and seeing what it was trying to do. The point is that pytest beautifully tells you where to look in the code (exactly which line, the backtrace, etc.) and why the assertion failed (e.g., did `res[0].initial_tablets > 0` fail because it was 0 or -7? The string "initial_tablets not configured" doesn't help anything in this process.

They don't hurt either, so I'm not against this, but I just believe this not helpful at best.